### ISSUE TITLE
NPC Movement

### ISSUE THREAD

**Issue by IllusionTheDev** [JitseB/NPCLib#166]
I wonder if it's possible to add some sort of movement, doesn't necessarily have to make a pathfinder.

**Comment by Mooselk** [comment]
Something like this? https://github.com/Mooselk/Spigot-CSR/blob/10bae8513124910835572130f9f304a9a4d62c85/CSR-Receiver/src/me/kate/receive/npcs/nms/v1_8_R3/NPC_v1_8_R3.java#L170

**Comment by IllusionTheDev** [comment]
Yeah, don't forget to do the distance check, as after a certain distance it needs to use the teleport packet.

**Comment by JitseB** [comment]
Closing issue. Movement packets will add a lot of work I am currently unwilling to invest. Though, feel free to make a PR. ;)

Also @Mooselk, although it is a-okay to modify and redistribute the NPCLib code - the MIT license requires you to properly address the original NPCLib license.
